primitive.cabal

```
@@ -44,6 +44,7 @@ Library
         Data.Primitive.Ptr
         Data.Primitive.MutVar
         Data.Primitive.MVar
+        Data.Primitive.PrimRef
```

```suggestion
        Data.Primitive.PrimVar
```
I just noticed that the file is still called `PrimRef.hs`.

---

@oberblastmeister you also need to rename the file.